ffmpeg/nodes.py

```diff
@@ -51,8 +52,21 @@ class FilterNode(Node):
     """FilterNode"""
     def _get_filter(self):
         params_text = self._name
-        arg_params = ['{}'.format(arg) for arg in self._args]
-        kwarg_params = ['{}={}'.format(k, self._kwargs[k]) for k in sorted(self._kwargs)]
+
+        # Helper function to escape uncomfortable characters
+        def escape_chars(seq, keys):
```

Comment:
I wonder if there's any way to encode unicode chars in ffmpeg args. You'd think there would be, since people might want to render Chinese text over a video